frontend/talentsearch/resources/js/components/awardDetailsForm/AwardDetailsForm.tsx

```diff
@@ -0,0 +1,106 @@
+import { Select } from "@common/components/form/Select";
+import { errorMessages } from "@common/messages";
+
+export const AwardDetailsForm: React.FunctionComponent = () => {
+  const intl = useIntl();
+
+  return (
+    <div>
+      <h2 data-h2-font-size="b(h3)">1. Award Details</h2>
+      <p>
+        Did you get recognized for going above and beyond? There are many ways
+        to get recognized, awards are just one of them.
+      </p>
+      <div data-h2-display="b(flex)" data-h2-padding="b(top, m)">
+        <div data-h2-padding="b(right, l)">
+          <Input
+            id="award-title"
+            label={intl.formatMessage({
+              defaultMessage: "Award Title",
+              description:
+                "Label displayed on award form for award title input",
+            })}
+            name="award-title"
+            type="text"
+            rules={{ required: intl.formatMessage(errorMessages.required) }}
+          />
+
+          <Select
+            id="awarded-to"
+            label={intl.formatMessage({
+              defaultMessage: "Awarded to",
+              description: "Label displayed on Award form for awarded to input",
+            })}
+            name="awarded-to"
+            nullSelection={intl.formatMessage({
+              defaultMessage: "Select a something...",
+              description:
+                "Null selection for something select input in the awarded to form.",
+            })}
+            rules={{
+              required: intl.formatMessage(errorMessages.required),
+            }}
+            options={[
+              { value: 1, label: "hi-1" },
+              { value: 2, label: "hi-2" },
+              { value: 3, label: "hi-3" },
+            ]}
+          />
+
+          <Input
+            id="issuing"
+            label={intl.formatMessage({
+              defaultMessage: "Issuing Organization or Institution",
+              description:
+                "Label displayed on award form for issuing organization input",
+            })}
+            name="issuing"
+            type="text"
+            rules={{ required: intl.formatMessage(errorMessages.required) }}
+          />
+
+          <Select
+            id="awarded-scope"
+            label={intl.formatMessage({
+              defaultMessage: "Award Scope",
+              description:
+                "Label displayed on Award form for award scope input",
+            })}
+            name="awarded-scope"
+            nullSelection={intl.formatMessage({
+              defaultMessage: "Select a something...",
+              description:
+                "Null selection for something select input in the award scope form.",
+            })}
+            rules={{
+              required: intl.formatMessage(errorMessages.required),
+            }}
+            options={[
+              { value: 1, label: "hi-1" },
+              { value: 2, label: "hi-2" },
+              { value: 3, label: "hi-3" },
```

Comment:
For this we want to show a list of departments, which should be passed down as a prop. So, there is 3 steps to this:
1) Create a AwardDetailsFormProps interface which defines the props and their types, and set it on the AwardDetailsForm component.
```js
import { Department } from "../../api/generated";

interface AwardDetailsFormProps {
  departments: Department[];
}

export const AwardDetailsForm: React.FunctionComponent<AwardDetailsFormProps> = () => {
  ...
}
```
2) From the list of departments map a list of options that can be added to the Select input. Here is an [example](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/talentsearch/resources/js/components/request/CreateRequest.tsx#L141-L152) that you can use.

3) Lastly, replace the options you have set with the departmentOptions

Reply:
Actually nvm don't do this 😅 I've been informed were not adding departments! However, if we were to add them, above is the way you would do it.

Reply:
I'll keep this in mind since I would have to ask how to do it!